src/wasm2c.c.tmpl

```diff
@@ -133,15 +222,49 @@ DEFINE_STORE(i64_store32, u32, u64);
 #define I32_TRUNC_U_F64(x) TRUNC_U(u32, f64, UINT32_MAX, >,  x)
 #define I64_TRUNC_U_F64(x) TRUNC_U(u64, f64, UINT64_MAX, >=, x)
 
-#define DEFINE_REINTERPRET(name, t1, t2)  \
-  static inline t2 name(t1 x) {           \
-    t2 result;                            \
-    memcpy(&result, &x, sizeof(result));  \
-    return result;                        \
+#define DEFINE_REINTERPRET(name, t1, t2)   \
```

Comment:
Some of these can be more efficient.
See:
https://github.com/microsoft/ChakraCore/pull/5890/files#diff-51806c1cfd597ef0db6e366e700bc6c7R199
https://github.com/microsoft/ChakraCore/pull/5890/files#diff-51806c1cfd597ef0db6e366e700bc6c7R210

Reply:
Good point, though I'd rather get this working and landed before optimizing. It still isn't quite there, unfortunately. It's pretty hard for me to test, since I don't normally run windows.